Demo/ImaginaryDemo/ImaginaryDemo/ViewController.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+import Fakery
+
+class ViewController: UITableViewController {
+
+  struct Constants {
+    static let imageWidth = 500
+    static let imageHeight = 500
+    static let imageNumber = 20
+  }
+
+  lazy var imaginaryArray: [NSURL] = {
+    let faker = Faker()
+    var array = [NSURL]()
+
+    for i in 0..<Constants.imageNumber {
+      if let imageURL = NSURL(
+        string: "http://lorempixel.com/\(Constants.imageWidth)/\(Constants.imageHeight)/?type=attachment&id=\(i)\(50)") {
```

Comment:
You can use faker.internet.image(Constants.imageWidth, Constants.imageHeight) + "?type=attachment&id=\(i)\(50)"